app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+def test_incorrect_year_format() -> None:
+    with pytest.raises(ValueError):
+        get_human_age(-12, 43.1)
```

Comment:
The `get_human_age` function expects integer inputs for both `cat_age` and `dog_age`. The test case should be adjusted to pass integer values. Consider using `get_human_age(-12, 43)` to test the ValueError for negative and non-integer inputs separately.